test/androidTest/java/org/smssecure/smssecure/util/Rfc5724UriTest.java

```diff
@@ -65,6 +65,7 @@ public void testGetSchema() throws Exception {
   public void testGetPath() throws Exception {
     final String[][] uriTestPairs = {
         {"sms:+15555555555",                      "+15555555555"},
+        //{"sms:%2B49555555555",                    "%2B555555555"},
```

Comment:
Commented because it fails. Can't test in TS because of https://github.com/WhisperSystems/TextSecure/issues/3761.